bug: ionic generate page seems to not work with angular cli 10

# Bug Report

Ionic CLI                     : 6.11.1 (#)
   Ionic Framework               : @ionic/angular 5.3.1
   @angular-devkit/build-angular : 0.1000.7
   @angular-devkit/schematics    : 10.0.7
   @angular/cli                  : 10.0.7
   @ionic/angular-toolkit        : 2.3.2

Capacitor:

   Capacitor CLI   : 2.4.0
   @capacitor/core : 2.4.0

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v10.16.3 (#)
   npm    : 6.14.8
   OS     : Windows 10



When trying to run  ionic g page home/examples (from the root directory of the project so ".../wrong/") I get the following error : 
Could not find an NgModule. Use the skip-import option to skip importing in NgModule.
[ERROR] Could not generate page.

The project is basically a blank project created by ionic start but just to be sure I have uploaded the project so that you can clone it for reproducibility: https://github.com/pooriaPoorsarvi/wrong

I think this may be an issue with the new Angular/Cli but since I was not sure and could not find anything on the internet about a problem with this specific version of angular and ionic I made the issue here.

Thank you for your time.

--- a/schematics/component/index.ts
+++ b/schematics/component/index.ts
@@ -130,13 +130,11 @@ export default function(options: ComponentOptions): Rule {
 
     const workspace = await getWorkspace(host);
     const project = workspace.projects.get(options.project);
-
-    let optPath = '';
     if (project && options.path === undefined) {
-      optPath = buildDefaultPath(project);
+      options.path = buildDefaultPath(project);
     }
 
-    const parsedPath = parseName(optPath, options.name);
+    const parsedPath = parseName(options.path as string, options.name);
     options.name = parsedPath.name;
     options.path = parsedPath.path;
     options.selector = options.selector ? options.selector : buildSelector(options, project?.prefix ?? 'app');

--- a/schematics/page/index.ts
+++ b/schematics/page/index.ts
@@ -186,14 +186,13 @@ export default function(options: PageOptions): Rule {
 
     const workspace = await getWorkspace(host);
     const project = workspace.projects.get(options.project);
-    let optPath = '';
     if (project && options.path === undefined) {
-      optPath = buildDefaultPath(project);
+      options.path = buildDefaultPath(project);
     }
 
     options.module = findRoutingModuleFromOptions(host, options);
 
-    const parsedPath = parseName(optPath, options.name);
+    const parsedPath = parseName(options.path as string, options.name);
     options.name = parsedPath.name;
     options.path = parsedPath.path;
     options.selector = options.selector ? options.selector : buildSelector(options, project?.prefix ?? 'app');

